packages/cli-platform-ios/src/tools/getProjectInfo.ts

```diff
@@ -0,0 +1,12 @@
+import execa from 'execa';
+import {IosProjectInfo} from '../types';
+
+export function getProjectInfo(): IosProjectInfo {
+  process.chdir('./ios');
```

Comment:
this is assuming we know where the ios folder is, which may not always be true, e.g. in monorepos. You should be able to use `config.platform.ios.sourceDir` instead